featuretools_ta1/multi_table.py

```diff
@@ -71,6 +71,11 @@ class Hyperparams(hyperparams.Hyperparams):
         semantic_types=['https://metadata.datadrivendiscovery.org/types/ControlParameter'],
         description="Should parsed columns be appended, should they replace original columns, or should only parsed columns be returned? This hyperparam is ignored if use_semantic_types is set to false.",
     )
+    max_features = hyperparams.Hyperparameter[int](
```

Comment:
can you add this to `hyperparameters_to_tune` list below? for single and multitable